browser/brave_screenshots/screenshots_tab_feature.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_BRAVE_SCREENSHOTS_SCREENSHOTS_TAB_FEATURE_H_
+#define BRAVE_BROWSER_BRAVE_SCREENSHOTS_SCREENSHOTS_TAB_FEATURE_H_
+
+#include <memory>
+
+#include "base/memory/weak_ptr.h"
+#include "chrome/browser/image_editor/screenshot_flow.h"
+
+namespace content {
+class DevToolsAgentHost;
+class DevToolsAgentHostClient;
+}  // namespace content
+
+namespace brave_screenshots {
+
+// This is a convenience method which enables us to defer attaching the tab
+// helper until the first request for a screenshot has been received.
+void TakeScreenshot(base::WeakPtr<content::WebContents>, int);
```

Comment:
are we allowed to omit parameter names in Chromium C++ now? I know you can in Blink land, but if so, awesome!

Reply:
Unfortunately, I'm not sure if you can - its mentioned as a difference to the Chromium/Google style:
https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/blink-c++.md#may-leave-obvious-parameter-names-out-of-function-declarations

Reply:
This was a pattern unfamiliar to me given my background, but I saw it here and there in the codebase and started experimenting with it myself 😀 That said, I'm quite uncomfortable with it, and prefer the explicitness of providing names up front. I'll stick with that approach instead.